Add Atlantic 2024 landing page (#2069)
writethedocs/www: docs/conf/atlantic/2024/news/welcome.rst

:template: {{year}}/generic.html

.. post:: January 4, 2024
   :tags: {{shortcode}}-{{year}}

Announcing Write the Docs {{ city }} {{year}}
===========================================

Welcome to the official launch of `Write the Docs {{ city }} {{year}} <https://www.writethedocs.org/conf/{{ shortcode }}/{{ year }}/>`_.

> We're excited to announce the dates for our next conference: **Write the Docs {{ city }} {{ year }} will be on {{ date.short }}**.

Reviewer: ```suggestion
We're excited to announce the dates for our next conference: **Write the Docs {{ city }} {{ year }} will be held on {{ date.short }}**.
```

Reviewer: I wonder if we should use the fully-spelled-out month name instead of the shortname for the month?

Following the success of last year's Atlantic conference, we will be running a **virtual event** again,
with a similar schedule.

The schedule will run from the
early afternoon to early evening in most European/African timezones,
and from early morning to late afternoon in US East Coast/South America timezones.
Of course, this being a virtual conference, attendees and speakers
are welcomed from anywhere in the world.

Sign up to the separate
`Atlantic conference mailing list <https://www.writethedocs.org/newsletter/>`_ to stay
up to date on the call for proposals, ticket sales and all other updates.

Call for Proposals
------------------

Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!

> **We plan to launch our CFP in early May 2024** for those who are excited to speak.

Reviewer: What do we think about opening the Atlantic CFP during Portland in April? Could be a good opportunity to announce and promote and schmooze about it earlier

Reviewer 2: yeah, I think this makes sense. Straightforward enough to set up, and because it's virtual, folks can more easily submit from Portland to Atlantic

> **We plan to launch our CFP in early May 2024** for those who are excited to speak.

Reviewer 2: ```suggestion
**We'll be launching the Call for Proposals early March 2024**, but if you're excited about speaking you can get thinking about your talk topic already.
```

> All talks will be pre-recorded again, so you'll be able to present from anywhere!

Reviewer 2: ```suggestion
We'll be asking speakers to upload a pre-recorded video of their talks, like last year, so you can present from anywhere!
```

> Start thinking about your talk proposal now, and we'll let you know as soon as the CFP opens.

Reviewer 2: ```suggestion
Start thinking about your talk proposal now, and we'll let you know via the usual channels as soon as the CFP opens.
```
